### DISCUSSION TITLE
如何解决因网络波动导致的补全文件缺失

### DISCUSSION THREAD

**Discussion by LF134** [Meloong-Git/PCL#3250] [提问]
来自rumengchuxing062
“应该是网络不稳定导致文件没有下全
BMCL API 和 MoJang API 均出现了无法解析网络名称，猜测应该是反复错误后终止补全”

**Comment by allMagicNB** [comment]
尝试更换DNS？

**Reply by LF134** [reply]
之后启动的时候好像都跳过了补全文件的过程了，能否通过删除残缺的补全文件使启动器重新下载

**Reply by allMagicNB** [reply]
> 之后启动的时候好像都跳过了补全文件的过程了，能否通过删除残缺的补全文件使启动器重新下载

应该行

**Reply by ghost** [reply]
版本下就留个json文件，然后把Library和assest都删了，rd /s /q %Temp%把缓存删了，然后重新启动游戏就能触发完整补全

**Reply by allMagicNB** [reply]
> 版本下就留个json文件，然后把Library和assest都删了，rd /s /q %Temp%把缓存删了，然后重新启动游戏就能触发完整补全

你这`rd /s /q %Temp%`是直接把Temp文件夹删了……

**Reply by ghost** [reply]
Temp也没有用啊。。。。就临时文件夹，难不成你往里面放国家机密么？

**Reply by allMagicNB** [reply]
> Temp也没有用啊。。。。就临时文件夹，难不成你往里面放国家机密么？

我是指，你把Temp文件夹也删了

**Reply by LF134** [reply]
重启电脑后删除library和assest就直接触发了补全下载，这次下载速度正常但还是发生了在接近百分之八十进度时直接完成并启动游戏，后续显示问题仍然存在

**Reply by WorldHim** [reply]
> 重启电脑后删除library和assest就直接触发了补全下载，这次下载速度正常但还是发生了在接近百分之八十进度时直接完成并启动游戏，后续显示问题仍然存在

不会是版本 json 的问题吧
尝试重新安装版本（存档、材质、配置等可以保留）

**Comment by LF134** [comment]


**Reply by allMagicNB** [reply]
你有地图存档啥的吗，备份一下直接删版本重新安装吧

**Reply by LF134** [reply]
没有，但是之前重装了整个版本和pcl2也没有解决

**Reply by allMagicNB** [reply]
> 没有，但是之前重装了整个版本和pcl2也没有解决

那你换DNS吧

**Reply by LF134** [reply]
感谢解答

**Reply by ghost** [reply]
我觉着没用

**Reply by allMagicNB** [reply]
> 我觉着没用

但是日志来看像是解析网址时出错
而解析的就是DNS

**Reply by ghost** [reply]
MoJang API 都解析这么多年了，总不可能有DNS没同步吧，这是自建还是古董啊
![image](https://github.com/Hex-Dragon/PCL2/assets/110439412/9e92e98d-b606-4927-b803-25887d735d64)
瞅见没，没网也是解析失败

**Reply by allMagicNB** [reply]
> MoJang API 都解析这么多年了，总不可能有DNS没同步，这是自建还是古董啊
> ![image](https://github.com/Hex-Dragon/PCL2/assets/110439412/9e92e98d-b606-4927-b803-25887d735d64)
> 瞅见没，没网也是解析失败

访问网页不也得请求DNS把域名转换为IP吗（

**Reply by ghost** [reply]
龙猫是瞎翻译的估计
这应该写成连接失败

**Comment by ghost** [comment]
> 来自rumengchuxing062 “应该是网络不稳定导致文件没有下全 BMCL API 和 MoJang API 均出现了无法解析网络名称，猜测应该是反复错误后终止补全”

WiFi链接的话建议换有线，不然就换个运营商
告诉我你的运营商信息，我可以想想怎么办（电信除外，因为世人皆知电信最稳定，如果电信都炸了那就没办法了）

**Reply by LF134** [reply]
应该是位置问题，距离WIFI直线距离十米隔了三面墙，问题在于之后跳过补全文件下载开启游戏，重装了版本和pcl还是没有解决

**Reply by ghost** [reply]
隔墙WIFI不建议用

**Comment by Pigeon0v0** [comment]
各网游加速器（UU 和迅游等）加速 MC 试试？

UU 加速器甚至加了一条提示 支持第三方启动器（

**Reply by ghost** [reply]
他已经尝试过VPN了，但是依旧无法下载

**Reply by allMagicNB** [reply]
> 他已经尝试过VPN了，但是依旧无法下载

鬼知道他用的啥VPN

**Reply by LF134** [reply]
bitznet核心访问，花钱的质量应该不会太差
